exercises/robot-simulator/canonical-data.json

```diff
@@ -213,10 +226,12 @@
       ]
     },
     {
-      "description": "moves the robot forward 1 space in the direction it is pointing",
+      "description": "Forward One",
+      "comments": [ "moves the robot forward 1 space in the direction it is pointing" ],
       "cases": [
         {
-          "description": "increases the y coordinate one when facing north",
+          "description": "facing north",
```

Comment:
```suggestion
          "description": "Move forward facing north increases y coordinate",
```

Reply:
Thats the same length as the original, which is pointless for me.  The goal is facilitate reasonable length unit test names.  And we already [know its moving forward from the parent](https://github.com/exercism/problem-specifications/blob/master/exercises/robot-simulator/canonical-data.json#L216-L219)